README.md

```diff
@@ -1,8 +1,120 @@
 
 # payments-email-verification
 
-This is a placeholder README.md for a new repository
+This service acts as a layer in front of the [email-verification](https://github.com/hmrc/email-verification) service to
+keep track of certain counts to prevent too many email verification attempts in certain scenarios. This service
+supports the management of email-verification passcode journeys.
 
-### License
+## Usage
+Ensure you have the `payments-email-verification` and configured in your application.conf:
+```
+microservices {
+   services {
+     payments-email-verification {
+       protocol = http
+       host = localhost
+       port = 10800
+      } 
+   }
+} 
+```
+and bring in `payments-email-verification-cor` as a library dependency to your service
+```
+"uk.gov.hmrc" %% "payments-email-verification-cor" % <version>
+```
+
+All endpoints are authenticated - a GG session must be present where the GG credentials can be retrieved.
+
+### Start an email verification passcode journey
+Given you have just captured an email address from the user, to start an email verification journey call the 
+start email verification journey endpoint by making use of the `PaymentsEmailVerificationConnector`:
+```scala
+import paymentsEmailVerification.connectors.PaymentsEmailVerificationConnector
+import paymentsEmailVerification.models.api.{StartEmailVerificationJourneyRequest, StartEmailVerificationJourneyResponse}
```

Comment:
Readme looks great and detailed, but do we need to explain the Request models - in particular using email/sensitive string? Or do consumers of this not need to worry?

Reply:
I put a link to the email-verification README where the parameters in the request are described there. It should basically be a 1-to-1 mapping between our case class and their's. For the email address, I made a choice here to make it not encrypted in the "public" API through the connector to make it easier to use - otherwise we'd have to worry about syncing up encryption tokens between this service and the calling service. The email is encrypted when it's stored in mongo though which is the main thing we need to worry about.

Reply:
Ok sounds good 😄